### DISCUSSION TITLE
Add timestamp using the attach function

### DISCUSSION THREAD

**Discussion by Barbapapazes** [adonisjs/core#2933] [Help] [answered]
Hello, 

Is-it possible to add timestamp in a pivot row using the attach function?

Thanks !

**Comment by arthurcvm** [comment]
Hi!
No, it's not possible, look the attach function:
```ts
attach(ids: (string | number)[] | Record<string, ModelObject>, trx?: TransactionClientContract): Promise<void>;
```
You can implements timestamp in the pivot passing `DateTime` from `luxon` in migration, or implements a id as PK and get the register to set a timestamp.

**Comment by thetutlage** [comment, marked as answer]
Pivot table timestamps are already supported (assuming you are talking about createdAt and updatedAt). Related docs https://docs.adonisjs.com/guides/models/relationships#pivot-table-timestamps.

**Reply by arthurcvm** [reply]
@thetutlage good!!!

**Reply by Barbapapazes** [reply]
hooo thanks a lot ! I discover new things in this docs every time ! 🥰
